models.js

```diff
@@ -7,4 +7,18 @@ exports.selectTopics = () => {
    
     return db.query(queryStr).then((result)=> {
         return result.rows})
+}
+
+exports.selectArticles = () => {
+
+    let queryStr = ` 
+    Select articles.* ,
```

Comment:
Nice use of subquery but it may be better to do a join and then do one query as it is better practice/more efficient.